package.json

```diff
@@ -31,6 +31,7 @@
     "got": "^7.1.0",
     "helmet": "^3.8.1",
     "inquirer": "^5.2.0",
+    "js-sha256": "^0.9.0",
```

Comment:
Did you check if none of the packages that are already included does not support `sha256`? As it is quite common hash function.

Reply:
yes, found alternative, thanks